bot/exts/halloween/spooknamerate.py

```diff
@@ -0,0 +1,325 @@
+
+                if winners:  # make sure there is a winner
+
+                    if len(winners) > 1:
+                        await channel.send(" and ".join([win[1]['author'].mention for win in winners]) + " !")
+                        score = winners[0][1]['score']
+                        # display winners, their words and scores
+                        await channel.send(f"Congratulations to all! You have a score of {score}!")
+                        wrds = ', '.join([f'{win[1]["word"]} ({win[1]["author"].mention})' for win in winners])
+                        await channel.send(f"Your words were: {wrds}!")
+
+                    else:  # there is only one winner
+                        # There cant be 0 winners since we are checking that above
+
+                        winner = winners[0][1]
+                        await channel.send(f"{winner['author'].mention}!")
+                        await channel.send(f"Congratulations {winner['author'].mention}! You have a score of "
+                                           f"{winner['score']}!")
+                        await channel.send(f"Your word was: **{winner['word']}**.")  # display winner, word and score
+
+                    # Send random party emojis
+                    party = [random.choice([':partying_face:', ':tada:']) for _ in range(random.randint(1, 10))]
+                    await channel.send(" ".join(party))
+
+                else:
+                    # no participants in current iteration
+                    await channel.send('Hmm... Looks like no one participated! :cry:')
+
+            self.messages = {}  # reset the messages
+
+        # send the next name
+        self.word = f"{random.choice(self.first_names)} {random.choice(self.last_names)}"
+
+        await channel.send("Let's move on to the next name!\nAnd the next name is...\n"
+                           f"**{self.word}**!\nTry to spookify that... :smirk:")
+
+        self.poll = False  # accepting responses
+
+    async def get_responses_list(self, final: bool = False) -> discord.Embed:
+        """Returns an embed containing the responses of the people."""
+        channel = await self.get_channel()
+
+        embed = discord.Embed(
+            color=Colours.red,
+            title="",
+            description=""
+        )
+
+        if self.messages:
+            if final:
+                embed.title = "Spook Name Rate is about to end!"
+                embed.description = ("This Spook Name Rate round is about to end in 2 hours! You can review "
+                                     "the entries below! Have you rated other's words?")
+            else:
+                embed.title = "All the spookified names!"
+                embed.description = "See a list of all the entries entered by everyone!"
+        else:
+            embed.title = "No one has added an entry yet..."
+
+        for message in self.messages:
+            data = self.messages.get(message)
```

Comment:
Looks like overcomplicating things,
```py
for message_id, data in self.messages.items()
```
would be simpler, as well as the variable would reflect the actual state of what it is